cwm_worker_operator/config.py

```diff
@@ -135,3 +135,4 @@
 REDIS_CLEANER_DELETE_ANY_HOSTNAME_ERROR_MIN_SECONDS = int(os.environ.get('REDIS_CLEANER_DELETE_ANY_HOSTNAME_ERROR_MIN_SECONDS') or '7200')
 
 WORKERS_CHECKER_SLEEP_TIME_BETWEEN_ITERATIONS_SECONDS = int(os.environ.get('WORKERS_CHECKER_SLEEP_TIME_BETWEEN_ITERATIONS_SECONDS') or '5')
+WORKERS_CHECKER_MAX_PARALLEL_DEPLOY_PROCESSES = int(os.environ.get('WORKERS_CHECKER_MAX_PARALLEL_DEPLOY_PROCESSES') or 2)
```

Comment:
Is `2` without quotes intentional?

Reply:
it works that way too, but I changed to string to be the same as other similar configurations